src/SourceDialog/SourceSelector.vala

```diff
@@ -51,13 +51,17 @@ public class Maya.View.SourceSelector : Gtk.Popover {
         var add_calendar_button = new Gtk.ModelButton ();
         add_calendar_button.text = _("Add New Calendar…");
 
+        var import_calendar_button = new Gtk.ModelButton ();
+        import_calendar_button.text = _("Import From ICS File…");
```

Comment:
I guess we should probably use the user-facing name of the file format and not the extension

```suggestion
        var import_calendar_button = new Gtk.ModelButton () {
            text = _("Import iCalendar File…")
        };
```

Reply:
Makes sense 👍🏼️